backend/.eslintignore

```diff
@@ -1 +1,2 @@
-node_modules/**
\ No newline at end of file
+node_modules/**
+test/**
```

Comment:
Why this?

Reply:
eslint was causing errors while trying to commit test files.

Reply:
as I was using console.logs in test file for better debugging

Reply:
you can remove them at the end before commiting but don't add it to eslint ignore

Reply:
ok